python/ray/_private/state.py

```diff
@@ -807,8 +807,9 @@ def get_node_to_connect_for_driver(self, node_ip_address):
             node_ip_address
         )
 
-    def get_draining_nodes(self) -> Set[str]:
-        """Get all the hex ids of nodes that are being drained."""
+    def get_draining_nodes(self) -> Dict[str, int]:
```

Comment:
Is this okay to break backward compatibilty like this?

Also, should we make it return a structured class instead of an integer just in case we will need additional information other than timeout?

Reply:
It's ok to break backward compatibility since the user is internal.

> Also, should we make it return a structured class instead of an integer just in case we will need additional information other than timeout?

I think we can do that when the need comes since we don't need to worry about backward compatibility